File: WebApplicationFSharp.Net/WebApplicationFSharp.Net/Areas/Customers/Models/CustomerViewModel.fs
namespace WebApplicationFSharp.Net.Customers.Models

open System
open Microsoft.FSharp.Linq.NullableOperators

type CustomerViewModel () =

    let _customer = Customer()

    member this.TxtName
        with get () = _customer.CustomerName
        and set value = _customer.CustomerName <- value

    member this.TxtAmount
        with get () = _customer.Amount.ToString()
        and set (value: string) = _customer.Amount <- Nullable <| Convert.ToDecimal(value)

    member this.CustomerLevelColor
        with get () =
            match _customer.Amount with
            | x when x ?> 2000.00M -> "red"
            | x when x ?> 1500.00M -> "orange"
            | _ -> "yellow"

File: WebApplication.Net/WebApplication.Net/Areas/Customers/Models/CustomerViewModel.cs
using System;

namespace WebApplication.Net.Customers.Models
{
    public class CustomerViewModel
    {
        private Customer _Customer = new Customer();

        public string TxtName
        {
            get { return _Customer.CustomerName; }
            set { _Customer.CustomerName = value; }
        }

        public string TxtAmount
        {
            get { return _Customer.Amount.ToString(); }
            set { _Customer.Amount = Convert.ToDecimal(value); }
        }

        public string CustomerLevelColor
        {
            get
            {
                if (_Customer.Amount > 2000.00M)
                {
                    return "red";
                }
                else if (_Customer.Amount > 1500.00M)
                {
                    return "orange";
                }
                else
                {
                    return "yellow";
                }
            }

        }

    }
}